src/components/RenderIndividualBook.jsx

```diff
@@ -1,46 +1,55 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { useDispatch } from 'react-redux';
-import { removeBook } from '../redux/books/booksSlice';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { removeBookFromApi, fetchApi } from '../redux/books/booksSlice';
 import styles from '../styles/renderIndividual.module.css';
 
-export default function RenderIndividualBook({ renderData }) {
+export default function RenderIndividualBook() {
   const dispatch = useDispatch();
+  const books = useSelector((state) => state.books);
+  useEffect(() => {
+    dispatch(fetchApi());
+  }, [dispatch]);
```

Comment:
Great work on dispatching the action after fetching the books from the `API` :+1: 
[OPTIONAL]
- [ ] In the dependency array, you are supposed to put in the value of a variable that if changed the `hook` will be called again, and the component re-render. YOu may consider set to an `[ ]` for now or `array of books you get from the store`

Reply:
Ok sure. Thank you